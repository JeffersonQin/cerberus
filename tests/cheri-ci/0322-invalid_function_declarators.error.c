int (*fun2(int arg, int x))(int x, int not_arg) {
  not_arg; // this is NOT in scope
}
