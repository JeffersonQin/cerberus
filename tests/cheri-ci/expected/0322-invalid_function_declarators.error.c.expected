cheri-ci/0322-invalid_function_declarators.error.c:2:3: error: use of undeclared identifier 'not_arg'
  not_arg; // this is NOT in scope
  ^
ยง6.5.1#2: 
2   An identifier is a primary expression, provided it has been declared as designating an
    object (in which case it is an lvalue) or a function (in which case it is a function
    designator).91)
